src/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -108,6 +108,7 @@ public void createDataSource() {
             if (workConf.getJdbcURL() != null && workConf.getJdbcURL().length()>0) {
               config.setJdbcUrl(workConf.getJdbcURL());
             }
+            config.setTransactionIsolation(workConf.getIsolationString());
```

Comment:
Is this needed for the normal connections that we create below?

Reply:
that method is used for DDL and load, so no